sccp/figures/figureThomsonDropPercents.py

```diff
@@ -9,56 +9,52 @@
 from ..imports import import_thomson
 import numpy as np
 from ..factorization import pf2
+from anndata import AnnData
+from matplotlib.axes import Axes
 import pandas as pd
 import seaborn as sns
+from scipy.stats import linregress
 
 
 def makeFigure():
     rank = 20
     data = import_thomson()
     ax, f = getSetup((2, 2), (1, 1))
-    bCellGeneSet = [
-        "PXK",
-        "MS4A1",
-        "CD19",
-        "CD74",
-        "CD79A",
-        "CD79B",
-        "BANK1",
-        "PTPRC",
-        "CR2",
-        "VPREB3",
-    ]
-
-    plot_weights_across_percents(
-        data, "B Cells", "CTRL4", 0, 1, 0.25, rank, bCellGeneSet, ax[0]
-    )
+
+    plot_weights_across_percents(data, "B Cells", "CTRL4", 0, 1, 0.25, rank, ax[0])
 
     ### Can add other cell types here
 
     return f
 
 
 def plot_weights_across_percents(
```

Comment:
Changed the function to now use r^2 instead of marker genes which we have been previously been using for the Drop figure